packages/@react-aria/tooltip/src/useTooltipTrigger.ts

```diff
@@ -132,7 +132,7 @@ export function useTooltipTrigger(props: TooltipTriggerProps, state: TooltipTrig
 
   return {
     triggerProps: {
-      'aria-describedby': state.open ? tooltipId : undefined,
+      'aria-describedby': state.isOpen ? tooltipId : undefined,
```

Comment:
https://github.com/adobe/react-spectrum/pull/1150 fixes this issue and includes a unit test.